release-notes/docs/3.9.0.md

```diff
@@ -0,0 +1,163 @@
+
+#### Cleanup operations may impact app load times
+
+We have implemented a purging function to clean up obsolete feedback and telemetry docs as part of [cht-core#5902](https://github.com/medic/cht-core/issues/5902). This will free up disk space and improve performance over time, however initially may cause slower app load times while the backlog is cleared. The cleanup is capped to 1000 docs per app load to limit the impact on users.
+
+### Supported software
+
+There are no required changes to the [supported software matrix](https://docs.communityhealthtoolkit.org/core/overview/supported-software/#dependencies) from `3.0.0`.
+
+| Node | CouchDB | Browsers | SMS bridge | Android | medic-android | medic-couch2pg |
+|----|----|----|----|----|----|---|
+| 8.11+ | 2.1+ | Chrome 53+, Firefox latest | medic-gateway | 4.4+ | 0.4.5+ | 3.0+ |
+
+## DHIS2 integration
+
+The CHT now supports calculating and exporting data for [DHIS2](https://www.dhis2.org) integration. The Core Framework can now be configured to calculate DHIS2 dataValues, aggregate patient level data, view aggregate data offline, and export the aggregate data formatted for DHIS2. The data is also available via an API so you can access it from an interoperability layer such as OpenHIM.
+
+For more information read [the documentation](https://docs.communityhealthtoolkit.org/apps/features/integrations/dhis2/).
+
+Issues: [cht-core#5661](https://github.com/medic/cht-core/issues/5661) and [cht-core#6002](https://github.com/medic/cht-core/issues/6002)
+
+## Supervise target progress
+
+Targets can now be configured to be aggregated and synced to allow supervisors to view the progress of their health workers towards their goals. These new "aggregate" targets show an overview of all workers and can be selected to show each worker's progress individually.
+
+Read [the documentation](https://docs.communityhealthtoolkit.org/apps/reference/targets/#targetsjs) to see how to configure this new aggregated target.
+
+Issues: [cht-core#4839](https://github.com/medic/cht-core/issues/4839) and [cht-core#4840](https://github.com/medic/cht-core/issues/4840)
+
+## RapidPro integration
+
+The Outbound Push feature has been extended to support submitting data to RapidPro so you can now trigger workflows based on changes in the CouchDB database.
```

Comment:
Interactive, structured messaging for health triage and clinical referrals is now possible with the integration of [RapidPro](https://rapidpro.io/). This opens new opportunities for a semi-automated, direct to patient approach to health assessments and care coordination at the community level. The Outbound Push feature has been extended to support submitting data to RapidPro so you can trigger workflows based on changes in the CouchDB database. 

Read the [feature overview](https://docs.communityhealthtoolkit.org/apps/features/integrations/rapidpro/) to learn more about this new integration.

^^placeholder link for documentation

Reply:
Linked to placeholder...